packages/atomic/src/components/result-template-components/atomic-result-fields-list/atomic-result-fields-list.tsx

```diff
@@ -11,7 +11,7 @@ import {ResultContext} from '../result-template-decorators';
 })
 export class AtomicResultFieldsList {
   @ResultContext()
-  private resizeObserver!: ResizeObserver;
+  private resizeObserver?: ResizeObserver;
```

Comment:
I'm not sure why `@ResultContext()` is applied to `resizeObserver`, that looks like a mistake. Do you think you could remove it?